src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -462,9 +462,9 @@ private void testDeleteCourseAction() {
         assertNotNull(BackDoor.getCourseAsJson(courseId));
         
         ______TS("delete course then proceed");
-        
+
         InstructorCoursesPage coursePage = courseEditPage.clickDeleteCourseLinkAndConfirm();
-        coursePage.verifyContains("Add New Course");
+        assertTrue(coursePage.getStatus().contains(String.format(Const.StatusMessages.COURSE_DELETED, courseId)));
```

Comment:
Is it possible to use stricter equality check (i.e `assertEquals(expected, coursePage.getStatus())`) rather than `assertTrue(coursePage.getStatus().contains(...))`?